Resources/public/webcomponents/ez-universal-discovery.html

```diff
@@ -0,0 +1,47 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<link rel="import" href="../../../webcomponents/ez-yui-app.html">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/finder.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/explorer.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/explorerlevel.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/search.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/selected.css">
+<link rel="import" href="mixins/ez-sandbox-yui-component.html">
+
+<style>
+    ez-universal-discovery {
+        display: block;
+        background: rgba(0, 0, 0, 0.4);
+
+        /* taken from universaldiscovery.css .ez-universaldiscovery-container */
+        position: fixed;
+        width: 100%;
+        height: 100%;
+        box-sizing: border-box;
+        padding: 2% 3%;
+        top: 0;
+        left: 0;
+    }
+
+    ez-universal-discovery .ez-view-universaldiscoveryview {
+        background: #fff;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+    }
+
+    ez-universal-discovery a {
+        cursor: pointer;
+    }
+
+    /* missing from modules/tabs.css (and coming for theme/modules/tabs.css) */
```

Comment:
I already commented about that in https://github.com/ezsystems/PlatformUIBundle/pull/876#discussion_r121621177